test/req.is.js

```diff
@@ -2,6 +2,7 @@
 
 var express = require('..')
 var request = require('supertest')
+var after = require('after')
```

Comment:
```suggestion
const after = require('after')
```
// in modern JS is better to use const instead of var!